packages/jellyfish-api-core/__tests__/category/misc/createMultiSig.test.ts

```diff
@@ -0,0 +1,103 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers/dist/index'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { wallet } from '@defichain/jellyfish-api-core'
+import { RpcApiError } from '../../../src'
+import { Testing } from '@defichain/jellyfish-testing'
+import { ECDH } from 'node:crypto'
+
+describe('create multi sig', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+  const testing = Testing.create(container)
+
+  beforeAll(async () => {
+    await container.start()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  async function getNewKey (walletType: wallet.AddressType): Promise<string> {
+    return (await testing.rpc.wallet.getAddressInfo(await testing.rpc.wallet.getNewAddress('', walletType))).pubkey
+  }
+
+  it('should create multi signature address 2-3', async () => {
+    let key0 = await getNewKey(wallet.AddressType.LEGACY)
+    key0 = ECDH.convertKey(key0, 'secp256k1', 'hex', 'hex', 'uncompressed').toString()
+    const key1 = await getNewKey(wallet.AddressType.LEGACY)
+    const key2 = await getNewKey(wallet.AddressType.LEGACY)
+    const keys = [key0, key1, key2]
+    const legacyAddr = (await client.misc.createMultiSig(2, keys, wallet.AddressType.LEGACY)).address
+    expect((await client.misc.createMultiSig(2, keys, wallet.AddressType.P2SH_SEGWIT)).address).toStrictEqual(legacyAddr)
+    expect((await client.misc.createMultiSig(2, keys, wallet.AddressType.BECH32)).address).toStrictEqual(legacyAddr)
+  })
+
+  it('should create multi signature address 2-2', async () => {
+    let key0 = await getNewKey(wallet.AddressType.LEGACY)
+    key0 = ECDH.convertKey(key0, 'secp256k1', 'hex', 'hex', 'uncompressed').toString()
+    const key1 = await getNewKey(wallet.AddressType.LEGACY)
+    const keys = [key0, key1]
+    const legacyAddr = (await client.misc.createMultiSig(2, keys, wallet.AddressType.LEGACY)).address
```

Comment:
```suggestion
    const { address } = await client.misc.createMultiSig(2, keys, wallet.AddressType.LEGACY)
```

Alternatively so you about parenthesis, which can be harder to read/review.